package.json

```diff
@@ -2,7 +2,7 @@
   "license": "MIT",
   "name": "open-foris-arena",
   "engines": {
-    "node": ">=10.13.0",
+    "node": "10.15.3",
```

Comment:
isn't better to use >=10.5.3 instead?

Reply:
I'm not sure; but i have changed it. see next commit